src/Mindee/Extraction/ExtractedImage.cs

```diff
@@ -0,0 +1,85 @@
+using System.IO;
+using Mindee.Input;
+using SkiaSharp;
+
+namespace Mindee.Extraction
+{
+    /// <summary>
+    /// An extracted sub-image.
+    /// </summary>
+    public class ExtractedImage
+    {
+        /// <summary>
+        /// SKBitmap wrapper for the image.
+        /// </summary>
+        public SKBitmap Image { get; }
+
+        /// <summary>
+        /// Name of the file.
+        /// </summary>
+        private string Filename { get; }
+
+        /// <summary>
+        /// String representation of the save format.
+        /// </summary>
+        private readonly string _saveFormat;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractedImage"/> class.
+        /// </summary>
+        /// <param name="image">The extracted image.</param>
+        /// <param name="filename">The filename for the image.</param>
+        /// <param name="saveFormat">The format to save the image.</param>
+        public ExtractedImage(SKBitmap image, string filename, string saveFormat)
+        {
+            this.Image = image;
+            this.Filename = filename;
+            this._saveFormat = saveFormat;
```

Comment:
Normally we don't add `this` when there is no confusion between the method and class variables.